### PR TITLE
Made changes to README.md as per the issue found here: https://github…

### PR DESCRIPTION
**Pull request by JamieBort**
….com/GSA/code-gov-front-end/issues/194

A similar PR may already be submitted!
Please search among the [Pull request](../) before creating one.

Thanks for submitting a pull request! Please provide enough information so that others can review your pull request:

For more information, see the `CONTRIBUTING` guide.

**Summary**

This PR fixes/implements the following **bugs/features**

* [ ] Bug 1
* [ ] Bug 2
* [ ] Feature 1
* [ ] Feature 2
* [ ] Breaking changes

Explain the **motivation** for making this change. What existing problem does the pull request solve?

**Test plan (required)**

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

**Code formatting**

**Closing issues**

Fixes #

### CODE REVIEW

**Comment by JamieBort** [comment]
This is one of my first pull requests; at least in a very long time. Please let me know if there is an issue with it. Or if you have any questions.

File: README.md [hunk 1]
```diff
@@ -97,6 +97,10 @@ For documentation on how to configure code-gov-front-end, go [here](CONFIGURATIO
 ## Questions?
 
 If you have questions, please feel free to [send us an email](mailto:code@gsa.gov).
+Twitter: @CodeDotGov
```
Comment (saracope) [inline README.md]: Can you add a link to the twitter account? Here's the markdown to use:
[@CodeDotGov](https://twitter.com/CodeDotGov)

Comment (saracope) [inline README.md]: Adding 2 spaces to the end of the line should make it force a line break. Try this on each of these lines you added.

File: README.md [hunk 2]
```diff
@@ -97,6 +97,10 @@ For documentation on how to configure code-gov-front-end, go [here](CONFIGURATIO
...
+Email: code@gsa.gov
+LinkedIn: code-gov
```
Comment (saracope) [inline README.md]: Please link out to the linkedin page. Here's the markdown you can use:
[code-gov](https://www.linkedin.com/company/code-gov/)

**Comment by saracope** [comment]
This was updated in another pull request. Thanks!

**Pull request closed without merging**